lightning/backend/local_unix.go

```diff
@@ -20,12 +20,29 @@ import (
 
 	"github.com/pingcap/errors"
 	"github.com/pingcap/failpoint"
+	"go.uber.org/zap"
+
+	"github.com/pingcap/tidb-lightning/lightning/log"
+)
+
+const (
+	// mininum max open files value
+	minRLimit = 1024
 )
 
+func GetSystemRLimit() (uint64, error) {
```

Comment:
add this function to `local_windows.go` too.

and maybe move `VerifyRLimit` out of `local_unix.go` back into `local.go`.

Reply:
done! `VerifyRLimit` is always called, so the current implementation is ok. Add I added a check if check-requirement is disabled, also skip check rlimit, to match the logic in local_windows.

Reply:
I mean since VerifyRLimit calls GetSystemRLimit, the former is no longer directly system-dependent. But this is fine too.